pkg/iss/iss.go

```diff
@@ -776,7 +761,8 @@ func (iss *ISS) initEpoch(newEpoch t.EpochNr) {
 
 	// Set the new epoch number and re-initialize list of orderers.
 	epoch := &epochInfo{
-		Nr: newEpoch,
+		Nr:         newEpoch,
+		Membership: iss.config.Membership, // TODO: Make a proper copy once reconfiguration is supported.
```

Comment:
> TODO: Make a proper copy once reconfiguration is supported

Do you plan to recycle the same ISS module instance for multiple configurations?

Reply:
Potentially, haven't decided yet. I think both options are viable and I'll see what is better only when actually writing the code implementing it. Since ISS itself is modular and basically only creates and destroys instances of sub-protocols, it might make sense to keep the same ISS module, especially since it keeps around data structures that are membership-independent (e.g. log of committed batches, request buckets). If it turns out to be simpler to completely delete the ISS module and create a new one, I'll do that.

For the time being, I'd keep this comment until the decision is made.